fitsio/test.py

```diff
@@ -525,6 +525,31 @@ def testHeaderFromCards(self):
+        a list of dicts
+        """
+
+        data="""SIMPLE  =                    T / file does conform to FITS standard             BITPIX  =                   16 / number of bits per data pixel                  NAXIS   =                    0 / number of data axes                            EXTEND  =                    T / FITS dataset may contain extensions            COMMENT   FITS (Flexible Image Transport System) format is defined in 'AstronomyCOMMENT   and Astrophysics', volume 376, page 359; bibcode: 2001A&A...376..359H CRAP    =  NaN                                                                  END                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             """
```

Comment:
flake8 is going to die on this string...

Reply:
can I just add `# noqa` after ?

Reply:
yup - or store the data as a file

Reply:
You can break it across lines too with `\`